stock_picking_package_preparation_line/models/stock_picking_package_preparation.py

```diff
@@ -180,16 +180,20 @@ def action_put_in_pack(self):
                         })
                 # Set the picking as "To DO" and try to set it as
                 # assigned
-                picking.action_confirm()
+                # skip_update_line_ids because picking is created based on
+                # preparation lines, updating lines would erase some fields
+                picking.with_context(
```

Comment:
wouldn't be better to update the context only one time by `picking = picking.with_context(skip_update_line_ids=True)`?